modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf/pojo.mustache

```diff
@@ -37,7 +37,9 @@ public class {{classname}} {{#parent}}extends {{{parent}}}{{/parent}} {
    * {{{description}}}  
   **/
 {{/description}}
-  private {{{datatypeWithEnum}}} {{name}} = {{{defaultValue}}};{{/vars}}
+  {{#isContainer}}private {{{datatypeWithEnum}}} {{name}}{{#required}} = {{{defaultValue}}}{{/required}}{{^required}} = null{{/required}};{{/isContainer}}
+  {{^isContainer}}private {{{datatypeWithEnum}}} {{name}} = {{{defaultValue}}};{{/isContainer}}
```

Comment:
Please do the following instead to avoid lines with empty spaces.
```
{{^isContainer}}
private {{{datatypeWithEnum}}} {{name}} = {{{defaultValue}}};
{{/isContainer}}
```

Reply:
Done